README.md

```diff
@@ -408,7 +408,7 @@ Plugins provide additional content and lets you customize rendered metrics.
 * **Core plugins**
   * [🗃️ Base content <sub>`base`</sub>](/source/plugins/base/README.md)
   * [🧱 Core <sub>`core`</sub>](/source/plugins/core/README.md)
-* **Github plugins**
+* **GitHub plugins**
```

Comment:
Brand

Reply:
[case-police](https://github.com/antfu/case-police) is a tool that can automatically check the correct casing of brands.